docs/components/tabs.md

```diff
@@ -396,6 +396,12 @@ export default {
 }
 </script>
 
+<style lang="scss" scoped>
+.horizontal-spacing {
```

Comment:
It looks like a utility class.
suggestion: rename it to `tab-1 `

Reply:
I have been renaming them like this in md files. Will leave these ones as is.